### PR TITLE
i18n: wire up strings

### PR DESCRIPTION
**Pull request by mimecuvalo**
finds all the strings in botcom and integrates them into the new system

### Change type

- [ ] `bugfix`
- [ ] `improvement`
- [ ] `feature`
- [ ] `api`
- [x] `other`

### CODE REVIEW

File: apps/dotcom/client/src/routes.tsx [hunk 1]
```diff
@@ -104,12 +110,7 @@ export const router = createRoutesFromElements(
 					lazy={() => import('./tla/pages/publish')}
 				/>
 				{/* Views that require login */}
-				<Route lazy={() => import('./tla/providers/RequireSignedInUser')}>
-					{/* User settings */}
-					<Route path={`/${PREFIX.tla}/profile`} lazy={() => import('./tla/pages/profile')} />
-					{/* Internal */}
-					<Route path={`/${PREFIX.tla}/debug`} lazy={() => import('./tla/pages/debug')} />
-				</Route>
+				<Route lazy={() => import('./tla/providers/RequireSignedInUser')}></Route>
```
Comment (MitjaBezensek) [inline apps/dotcom/client/src/routes.tsx:113]: There's nothing inside here, do we want to keep it for any future routes?

File: apps/dotcom/client/src/tla/components/TlaErrorContent/TlaErrorContent.tsx [hunk 2]
```diff
@@ -46,7 +44,7 @@ export function TlaErrorContent({ error }: { error: TlaPageErrorType }) {
 			return (
 				<div className={styles.container}>
 					<p className="tla-text_ui__regular">
-						{raw('Sorry, you don’t have access to that workspace.')}
```
Comment (MitjaBezensek) [inline apps/dotcom/client/src/tla/components/TlaErrorContent/TlaErrorContent.tsx:49]: When does this error occur?

File: apps/dotcom/client/src/tla/components/TlaFileShareMenu/TlaFileShareMenu.tsx [hunk 3]
```diff
@@ -70,9 +69,17 @@ export function TlaFileShareMenu({
 					{app ? (
 						<TlaTabsRoot activeTab={shareMenuActiveTab} onTabChange={handleTabChange}>
 							<TlaTabsTabs>
-								<TlaTabsTab id="share">{raw('Invite')}</TlaTabsTab>
-								<TlaTabsTab id="export">{raw('Export')}</TlaTabsTab>
-								{showPublishTab && <TlaTabsTab id="publish">{raw('Publish')}</TlaTabsTab>}
+								<TlaTabsTab id="share">
+									<F defaultMessage="Invite" />
```
Comment (MitjaBezensek) [inline apps/dotcom/client/src/tla/components/TlaFileShareMenu/TlaFileShareMenu.tsx:73]: Just as a curiosity, why `F`?

File: apps/dotcom/client/src/tla/components/TlaSidebar/TlaSidebar.tsx [hunk 4]
```diff
@@ -82,14 +90,19 @@ export const TlaSidebar = memo(function TlaSidebar() {
 })
 
 function TlaSidebarWorkspaceLink() {
-	const raw = useRaw()
+	const brandName = 'tldraw'
+	// const intl = useIntl()
```
Comment (MitjaBezensek) [inline apps/dotcom/client/src/tla/components/TlaSidebar/TlaSidebar.tsx:94]: Let's remove this?

**Review by MitjaBezensek — APPROVED** [review]

File: apps/dotcom/client/src/routes.tsx (hunk 1 above)
Comment (mimecuvalo) [reply apps/dotcom/client/src/routes.tsx:113]: yeah, i'm just keeping it around as a stub for future routes

File: apps/dotcom/client/src/tla/components/TlaErrorContent/TlaErrorContent.tsx (hunk 2 above)
Comment (mimecuvalo) [reply apps/dotcom/client/src/tla/components/TlaErrorContent/TlaErrorContent.tsx:49]: I'm not exactly sure, but I think David stubbed it out for future groups?

File: apps/dotcom/client/src/tla/components/TlaFileShareMenu/TlaFileShareMenu.tsx (hunk 3 above)
Comment (mimecuvalo) [reply apps/dotcom/client/src/tla/components/TlaFileShareMenu/TlaFileShareMenu.tsx:73]: it's short for `FormattedMessage` which, I dunno, just seems really long/a mouthful.

File: apps/dotcom/client/src/tla/components/TlaSidebar/TlaSidebar.tsx (hunk 4 above)
Comment (mimecuvalo) [reply apps/dotcom/client/src/tla/components/TlaSidebar/TlaSidebar.tsx:94]: oops! thank you

**Pull request merged**